testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
 {
+    "com.aisquared.zoomtext": {
+        "name": "ZoomText",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configuration": {
+                "type": "gpii.settingsHandlers.INISettingsHandler",
+                "liveness": "manualRestart",
+                "options": {
+                    "filename": "${{environment}.AppData}\\Freedom Scientific\\ZoomText\\2019\\Config\\zten-US.zxc",
```

Comment:
This is probably gonna need to be updated to 2020.

Reply:
You would think so, but we should probably check with the pilot/case studies team on this...